<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Devices</class>
 <widget class="QWidget" name="Devices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="devices_list">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>256</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
